virocon/contours.py

```diff
@@ -480,8 +661,35 @@ def cell_averaged_pdf(self, dist_idx, coords):
         fbar_out = fbar.reshape(fbar_out_shape)
         return fbar_out / dx
 
-
 class DirectSamplingContour(Contour):
+    """
+    Direct sampling contour as introduced by Huseby et al. (2013) [1]_
+
```

Comment:
Our DirectSampling contour only works for 2D models. We should make this more clear in the docstring. 

also attributes, see IFORM comment